scripts/config_gen.php

```diff
@@ -304,11 +301,6 @@ function write_config_file($settings, $filters) {
     Hm_Output_Modules::try_queued_modules();
     Hm_Output_Modules::process_all_page_queue();
     Hm_Output_Modules::try_queued_modules();
-    $settings['handler_modules'] = Hm_Handler_Modules::dump();
-    $settings['output_modules'] = Hm_Output_Modules::dump();
-    $settings['input_filters'] = $filters;
```

Comment:
Previous hm3.rc json file had these 3 arrays - handler_modules, output_modules and input_filters generated automatically from config_gen. Now we have dynamic config, how do we generate these 3 arrays? I see lib/modules_exec.php setup_production_modules method uses the 3 arrays from config but I am not sure we have them there with the new setup. Maybe generate another file in config folder with these 3 arrays when doing config_gen?

Reply:
a dynamic file dynamic.php file will now be created when doing config_gen.

Reply:
Alright. However, I'd prefer it to be in the .gitignore list, so it doesn't end up in the repo.